src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,26 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.task.Task;
+import duke.tasklist.Tasklist;
+import duke.ui.Ui;
+
+public class AddCommand extends Command {
+    private Task task;
+
+    // Constructor
+    private AddCommand(Task task) {
+        this.task = task;
+    }
+
+    public static Command createAddCommand(Task task) {
+        return new AddCommand(task);
+    } // End method.
```

Comment:
I like that you use comment to state the end of a method. It might be useful when all methods are long, but for short methods, it seems redundant to me. 

Also, if you use for 1 method, you probably need to include for all, else it will look inconsistent, no? In my opinion, I won't include it to make my life easier. :)